CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Upcoming
+
+* Strip down forms sass to be simpler and more useful
```

Comment:
I don't know if I can be more descriptive about it without over explaining. Basically we never used this file, these changes are removing most of what was there and adding a few things that we are more likely to be useful when added to a project